aspnetcore/signalr/publish-to-azure-web-app.md

```diff
@@ -1,63 +1,80 @@
 ---
-title: Publish an ASP.NET Core SignalR app to Azure Web App
+title: Publish an ASP.NET Core SignalR app to Azure App Service
 author: bradygaster
-description: Publish an ASP.NET Core SignalR app to Azure Web App
+description: Learn how to publish an ASP.NET Core SignalR app to the Azure App Service.
 monikerRange: '>= aspnetcore-2.1'
 ms.author: bradyg
 ms.custom: mvc
-ms.date: 04/20/2018
+ms.date: 06/25/2019
 uid: signalr/publish-to-azure-web-app
 ---
+# Publish an ASP.NET Core SignalR app to Azure App Service
 
-# Publish an ASP.NET Core SignalR app to an Azure Web App
+By [Brady Gaster](https://twitter.com/bradygaster)
 
-[Azure Web App](/azure/app-service/app-service-web-overview) is a [Microsoft cloud computing](https://azure.microsoft.com/) platform service for hosting web apps, including ASP.NET Core.
+[Azure App Service](/azure/app-service/app-service-web-overview) is a [Microsoft cloud computing](https://azure.microsoft.com/) platform service for hosting web apps, including ASP.NET Core.
 
 > [!NOTE]
-> This article refers to publishing an ASP.NET Core SignalR app from Visual Studio. Visit [SignalR service for Azure](https://azure.microsoft.com/services/signalr-service) for more information about using SignalR on Azure.
+> This article refers to publishing an ASP.NET Core SignalR app from Visual Studio. For more information, see [SignalR service for Azure](https://azure.microsoft.com/services/signalr-service).
```

Comment:
This is confusing -- the "from Visual Studio" clause leads me to expect something related to that, like an alternative publish method, perhaps (which the next paragraph does).  I would move this to the Additional resources section at the end.

Reply:
I'll defer to @bradygaster ... he may have a special desire to call this out here.